pkg/migrations/mls/20250310145526_upgrade-identity.up.sql

```diff
@@ -0,0 +1,22 @@
+SET
+    statement_timeout = 0;
+
+DROP INDEX idx_address_log_address_inbox_id;
+
+ALTER TABLE address_log
+ADD COLUMN identifier_kind INT;
+
+ALTER TABLE address_log
+RENAME COLUMN address TO identifier;
+
+-- Default all of the existing identifier_kinds to 1 (Ethereum)
+UPDATE address_log
+SET
+    identifier_kind = 1;
+
+ALTER TABLE address_log
+ALTER COLUMN identifier_kind
+SET
+    NOT NULL;
+
+CREATE INDEX idx_address_log_identifier_inbox_id ON address_log (identifier, identifier_kind, inbox_id);
```

Comment:
@neekolas Do you think removing and adding a new index on this table will be an issue?

Reply:
I don't think it's a huge deal because the table is relatively small. But it will block the node from coming up until it's completed.

Would be better if we could add the index concurrently, but IIRC we get errors doing that because it happens in a transaction

Reply:
From my recollection we can't use it in our migrations because the migration runner wraps everything in a transaction, and transactions can't create indexes concurrently. But my knowledge of this codebase is very out of date and maybe the issue is resolved.

You can try it. If the issue is still there every test that touches the DB will fail.